CalculatorTests/CalculatorTests/CalculatorTests.cs

```diff
@@ -0,0 +1,233 @@
+﻿using System;
+using NUnit.Framework;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Chrome;
+
+namespace CalculatorTests
+{
+    public class CalculatorTests
+    {
+        private IWebDriver driver;
+        private IWebElement loginfld;
+        private IWebElement passwordfld;
+        private IWebElement loginBtn;
+
+        [SetUp]
+        public void Setup()
+        {
+            driver = new ChromeDriver();
+            driver.Manage().Timeouts().PageLoad = TimeSpan.FromSeconds(60);
+            driver.Manage().Timeouts().ImplicitWait = TimeSpan.FromSeconds(10);
+            driver.Url = "http://127.0.0.1:8080/Login";
+            loginfld = driver.FindElement(By.Id("login"));
+            passwordfld = driver.FindElement(By.Id("password"));
+            loginBtn = driver.FindElement(By.Id("loginBtn"));
+            loginfld.SendKeys("test");
+            passwordfld.SendKeys("newyork1");
+            loginBtn.Click();
+        }
+
+        [TearDown]
+        public void TearDown()
+        {
+            driver.Quit();
+        }
+
+        [TestCase("", "", "")]
+        [TestCase("0", "0", "0")]
+        [TestCase("0", "10", "365")]
+        [TestCase("0", "0", "365")]
+        public void EmptyOrZeroDepositTest(string depositAmount, string roi, string investmentTerm)
+        {
+            // arrange
+            IWebElement amount = driver.FindElement(By.Id("amount"));
+            IWebElement percent = driver.FindElement(By.Id("percent"));
+            IWebElement term = driver.FindElement(By.Id("term"));
+            IWebElement d365 = driver.FindElement(By.Id("d365"));
+
+        // act
+        amount.SendKeys(depositAmount);
```

Comment:
Please move the lines one tab right.

```suggestion
                // act
                amount.SendKeys(depositAmount);
```

Reply:
I don't understand why I need to move these 2 lines fo a 1 tab in a right?

`

            // arrange
            IWebElement amount = driver.FindElement(By.Id("amount"));
            IWebElement percent = driver.FindElement(By.Id("percent"));
            IWebElement term = driver.FindElement(By.Id("term"));
            IWebElement d365 = driver.FindElement(By.Id("d365"));

            // act
            amount.SendKeys(depositAmount);
            percent.SendKeys(roi);
            term.SendKeys(investmentTerm);
            d365.Click();

            // assert
            string expectedIncome = "0.00";
            string expectedInterest = "0.00";
            string actualIncome = driver.FindElement(By.Id("income")).GetAttribute("value");
            string actualInterest = driver.FindElement(By.Id("interest")).GetAttribute("value");
`